### PR TITLE
Update _visitor_events.md

### PR DESCRIPTION
**Pull request by ihtnayaj**

### CODE REVIEW

File: source/includes/_visitor_events.md [hunk 1]
```diff
@@ -8,11 +8,9 @@
 |---|---|
 |GET|/client_reports/visitor_events?[query_params]|
 
-This API can be used to retrieve capture events for a list of visitor_ids for a gmaid.
+This API should be used to retrieve all events for a visitor_ids for a given advertiser (GMAID). The response will contain a chornological list of events. It can also be optionally grouped by visitor_id. You can also request for events within a data range.
```
Comment (clarkbremer) [inline source/includes/_visitor_events.md:11]: "a list of visitor_ids"

Comment (clarkbremer) [reply source/includes/_visitor_events.md:11]: Last sentence is not grammatically correct.  Suggest: "Events can also be restricted to a date range".

File: source/includes/_visitor_events.md [hunk 2]
```diff
@@ -197,22 +195,22 @@ https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&vi
...
 |---|---|---|
-|event_id|Event ID|
-|visitor_id|Visitor ID|
-|event_time|Event Time|
-|event_type|Event Type|
-|event_subtype|Event Subtype|
-|referrer_source|Referrer Source|
-|referrer_type|Referrer Type|
-|page_url|Page URL|
+|event_id|Event ID|An id for the event and unique to every event
```
Comment (clarkbremer) [inline source/includes/_visitor_events.md:206]: Actually, not unique to every event.  I think if they are within a certain small time span, it's the same.  "same visit" perhaps?

File: source/includes/_visitor_events.md [hunk 3]
```diff
@@ -197,22 +195,22 @@ https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&vi
...
+|visitor_id|Visitor ID|An id of a visitor to an advertiser’s site for a given browser & device. The same visitor will have a different id if a different browser on the same device or a different device is used.
```
Comment (clarkbremer) [inline source/includes/_visitor_events.md:207]: All the rows in the table should end with a `|` to make it render corectly.

**Review by clarkbremer — CHANGES_REQUESTED** [review]

File: source/includes/_visitor_events.md (hunk 2 above)
Comment (brianroth-rl) [reply source/includes/_visitor_events.md:206]: event_id is a GUID and is unique. In fact, it's the primary key of the events table.

Comment (brianroth-rl) [reply source/includes/_visitor_events.md:206]: need a | at the end here also

Comment (clarkbremer) [reply source/includes/_visitor_events.md:206]: [event_id + occurrence_time] are the unique primary key.  There are lots of duplicate event_ids in the table.

File: source/includes/_visitor_events.md [hunk 4]
```diff
@@ -229,3 +227,49 @@ https://api.reachlocalservices.com/client_reports/visitor_events?gmaid=TEST_1&vi
...
+Describes the type of domain the visitor came from.
+| Field Name |Description |
+|---|---| 
+|Search|Google.com, lycos.com etc.|
+|Social|pinterest.com, plus.url.google etc.|
+|Directory|whitepages.com, superpages.com, birkenheadpages.co.uk etc.|
+|Other|allstarrents.com, maac.com etc.|
+|Direct|Visitor typed the URL|
+|unavailable|Cannot be determined|
+|
```
Comment (brianroth-rl) [inline source/includes/_visitor_events.md:275]: this | looks out of place

File: source/includes/_visitor_events.md [hunk 5]
```diff
@@ -8,11 +8,9 @@
 |---|---|
 |GET|/client_reports/visitor_events?[query_params]|
 
-This API can be used to retrieve capture events for a list of visitor_ids for a gmaid.
+This API should be used to retrieve all events for a list of visitor_ids for a given advertiser (GMAID). The response will contain a chornological list of events. It can also be optionally grouped by visitor_id. Events can also be restricted to a date range.
```
Comment (brianroth-rl) [inline source/includes/_visitor_events.md:11]: chornological is spelled incorrectly

**Review by brianroth-rl — CHANGES_REQUESTED** [review]

**Review by brianroth-rl — APPROVED** [review]

**Review by clarkbremer — APPROVED** [review]

**Pull request merged**